src/garage/torch/modules/mlp_module.py

```diff
@@ -56,6 +56,12 @@ def __init__(self,
                          output_nonlinearity, output_w_init, output_b_init,
                          layer_normalization)
 
+        self.input_dim = input_dim
```

Comment:
can you make these _private?

Reply:
I need to be able access output_dim outside. Should I just add a getter function?